spec/factories/kudos.rb

```diff
@@ -2,12 +2,8 @@
   factory :kudo do
     title { 'Testtitle' }
     content { 'fcontent' }
-    if Employee.find_by(email: 'test2338@test.com')
-      giver { Employee.find_by(email: 'test2338@test.com') }
-      receiver { Employee.find_by(email: 'test2484@test.com') }
-    else
-      giver { Employee.create!(email: 'test2338@test.com', password: 'password') }
-      receiver { Employee.create!(email: 'test2484@test.com', password: 'password') }
-    end
+    giver { create(:employee) }
+    receiver { create(:employee) }
+    company_value { create(:company_value) }
```

Comment:
in this case you can also simply write:
```suggestion
    company_value
```